mmv1/templates/terraform/pre_delete/compute_region_network_endpoint.go.tmpl

```diff
@@ -27,6 +27,7 @@ if fqdnProp != "" {
   toDelete["fqdn"] = fqdnProp
 }
 
+<% unless version == 'ga' -%>
```

Comment:
It looks like this reverts a PR that was made before the Ruby -> Go rewrite, so you'll need to manually convert all these version guards to the use go template equivalents - see https://googlecloudplatform.github.io/magic-modules/reference/ruby-go-changes/#version-guards for more information. It should be a trivial change but let me know if you'd rather have me do it.